scattering/tests/test_van_hove.py

```diff
@@ -78,3 +85,64 @@ def test_self_warning(parallel):
             self_correlation=True,
             parallel=parallel,
         )
+
+        
+def test_vhf_from_pvhf():
+    trj = md.load(get_fn("spce.xtc"), top=get_fn("spce.gro"))
+    unique_atoms = get_unique_atoms(trj)
+    tuples_combination = combinations_with_replacement(unique_atoms, 2)
+
+    # obtaining g_r_t from total func
+    chunk_length = 20
+    r, t, g_r_t = compute_van_hove(trj, chunk_length=chunk_length)
+
+    # obtating dict of np.array of pvhf
+    partial_dict = {}
+
+    for pairs in tuples_combination:
+        pair1 = pairs[0]
+        pair2 = pairs[1]
+        if pairs[0].name > pairs[1].name:
+            pair2 = pairs[0]
+            pair1 = pairs[1]
+       
+        x = compute_partial_van_hove(
+            trj,
+            chunk_length=chunk_length,
+            selection1=f"name {pair1.name}",
+            selection2=f"name {pair2.name}",
+        )
+        partial_dict[pairs] = x[1]
+
+    # obtaining total_grt from partial
+    total_g_r_t = vhf_from_pvhf(trj, partial_dict)
+
+    assert np.allclose(g_r_t, total_g_r_t)
+
+trj = md.load(get_fn("spce.xtc"), top=get_fn("spce.gro"))
```

Comment:
I'm not a fan of defining variables outside of the functions, but I understood why you did so in this case.

Instead, could you split these three cases into three separate tests without using `@pytest.mark.parametrize`?  Further you can add a `match` argument when running `pytest.raises` like:
```
with pytest.raises(ValueError, match="Must be a tuple"):
    vhf_from_pvfh()
```
That way the test is more explicit in the ValueError it's checking for.